code/game/machinery/computer/cloning.dm

```diff
@@ -485,7 +485,7 @@
 	R.fields["id"] = copytext_char(md5(mob_occupant.real_name), 2, 6)
 	R.fields["UE"] = dna.unique_enzymes
 	R.fields["UI"] = dna.uni_identity
-	R.fields["SE"] = dna.struc_enzymes
+	R.fields["SE"] = dna.mutation_index
```

Comment:
By the by, this will be displayed as "\list" on the UI since this is a list, please fix that one way or another.